README.md

```diff
@@ -172,47 +177,47 @@ To understand how to query the local API review the Swagger definition here: [ht
 ![Swagger Definition](docs-images/swagger.JPG)
 
 To test the API, make a new postman request:
+
 * POST
-* URL: http://localhost:5000/text/analytics/v2.0/sentiment
+* URL: <http://localhost:5000/text/analytics/v2.0/sentiment>
```

Comment:
should the '>' values be added or is this confusing for what you actually need to enter into postman?

Reply:
its a markdown rule requirement since it was flagging this as naked URL, it autocorrected this to the above format. Perhaps we can use the approved [] () format.

Reply:
ah I see - yes great spot lets use the formal format of []()